hal/src/common/thumbv7em/sercom/uart.rs

```diff
@@ -156,7 +156,8 @@ macro_rules! uart {
                             w.runstdby().set_bit(); // Run in standby
                             w.form().bits(0); // 0 is no parity bits
 
-                            // w.mode().usart_int_clk(); // Internal clock mode
+                            // TODO: does this need to be commented out to work on same54?
+                            w.mode().usart_int_clk(); // Internal clock mode
```

Comment:
This line was commented out in the SAME54 implementation, but not in the SAMD51 implementation. Any ideas on why this was this case?